### DISCUSSION TITLE
Repo cloning error in PE+ 13 for vayu

### DISCUSSION THREAD

**Discussion by sanjeevstunner** [ROM-builders/temporary#19404] [General]
https://cirrus-ci.com/task/6278037734424576?logs=build#L36

IDK why its fetching from pixel devices remote. This only happens with PE+, regular PE works just fine. And this is happening for a long time now. Can you please find a way to fix this 😔

**Comment by Apon77** [comment]
1. Your local manifest have problem duplicate device path
2. Vendor folder had problem

**Comment by sanjeevstunner** [comment]
https://github.com/sanjeevstunner/Manifest/blob/vayu_aosp/local_manifest.xml

No, its fine 

Also, vendor fine too 
https://github.com/sanjeevstunner/android_vendor_xiaomi_vayu/blob/13/vayu-vendor.mk

Look

**Reply by sanjeevstunner** [reply]
@Apon77
